libcxx/include/__memory/unique_ptr.h

```diff
@@ -410,7 +410,7 @@ class _LIBCPP_UNIQUE_PTR_TRIVIAL_ABI _LIBCPP_TEMPLATE_VIS unique_ptr<_Tp[], _Dp>
 public:
   typedef _Tp element_type;
   typedef _Dp deleter_type;
-  using pointer = __pointer<_Tp, deleter_type>;
+  using pointer _LIBCPP_NODEBUG = __pointer<_Tp, deleter_type>;
```

Comment:
Same here.

Reply:
"Same" as which comment?

Reply:
About applying to public API typedefs. I think we resolved that question above though, you can drop `_LIBCPP_NODEBUG` from all public API typedefs.